source/includes/options-mongosqld.yaml

```diff
@@ -32,8 +32,32 @@ name: verbose
 directive: option
 aliases: -v
 description: |
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  Can be specified multiple times or with an integer to provide greater
```

Comment:
`This option can...`

Reply:
Actually, I would prefer "You can provide this flag repeatedly or with an integer...`